fabric/dataclient.go

```diff
@@ -446,13 +446,32 @@ func convertOne(fg *Fabric) ([]*eskip.Route, error) {
 				allowedServices := decideAllowedServices(fg.Spec.AllowList, m.AllowList)
 				if len(allowedServices) > 0 || allowedServices == nil {
 					r := createServiceRoute(m, eskipBackend, allowedOrigins, allowedServicesToFilterArgs(allowedServices), privs, fg.Metadata.Name, fg.Metadata.Namespace, host, p.Path)
+					if m.Response != nil {
```

Comment:
`m.Response` can be passed to the function (`patchRouteWithStaticResponse`) and the `if` check can be incorporated inside the function.

Reply:
done